### ISSUE TITLE
testing 5377: missing/ different focus

### ISSUE THREAD

**Issue by CarolingerSeilchenspringer** [liqd/a4-meinberlin#4199]
**URL:** /projekte
**user:** anyone without mouse
**expected behaviour:** can see focus
**behaviour:** cannot see focus or having a dot instead of frame for  level of participation and status
**important screensize:**
**device & browser:** monetery chrome/ safari/ edge: no focus, firefox: dot  (see screenshot)
**Comment/Question:** 

Screenshot?
<img width="587" alt="Bildschirmfoto 2022-02-24 um 16 56 37" src="https://user-images.githubusercontent.com/35491681/155561977-e5959616-e012-4a70-aa9e-d00a0110747c.png">
